source/main/gameplay/BeamEngine.cpp

```diff
@@ -196,30 +235,34 @@ void BeamEngine::update(float dt, int doUpdate)
 		totaltorque -= 8.0f * hydropump / (curEngineRPM * 0.105f * dt);
 	}
 
-	if (running && contact && curEngineRPM < (maxRPM * 1.25f) && curEngineRPM > stallRPM)
+	if (running && contact && curEngineRPM < (maxRPM * 1.25f))
 	{
 		totaltorque += getEnginePower(curEngineRPM) * acc;
 	}
 
-	if (running && curEngineRPM < stallRPM)
+	if (!is_Electric)

```

Comment:
I feel it would make more sense to set the stallRPM to 0.0 for electric motors instead of a hard check for is_Electric, it would be more flexible.